README.md

```diff
@@ -1,2 +1,26 @@
 # chart-postgresql
+[![Build Status](https://dev.azure.com/hmcts/PlatformOperations/_apis/build/status/chart-postgresql)](https://dev.azure.com/hmcts/PlatformOperations/_build?definitionId=856)
 This chart is intended for adding the postgres sql databases.
+
+## Prerequisites
+
+- To use this chart, you need a postgres Flexible server created beforehand.
+- Please see [this](https://github.com/hmcts/cnp-flux-config/blob/master/apps/sscs/preview/aso/sscs-postgres.yaml) link where the Flexible server is deployed using ASO.
+
+## Example configuration
+
+```yaml
```

Comment:
this isn't really an example it's reference, it would also be good to supply a full example

Reply:
sure